fission/environment.go

```diff
@@ -90,6 +90,11 @@ func envCreate(c *cli.Context) error {
 		}
 	}
 
+	extractArchive := true
```

Comment:
Usage in fission/main.go says "defaults to false", but here is true.
And I will suggest setting `Value` field in cli.BoolFlag instead of hard-code here.